libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -172,7 +172,7 @@ class YulUtilFunctions
 
 	/// @returns the name of a function that resizes a storage array
```

Comment:
Please document the semantics for statically-sized arrays.

Reply:
I added comment that explains what happens for statically-sized arrays, hope it is good enough